clis/dtcli/src/dtcli.rs

```diff
@@ -0,0 +1,113 @@
+/// Code for the daemon which runs actual libtelio in the background.
+use anyhow::Result;
+
+use tokio::task::JoinHandle;
+
+use telio_task::io::{wait_for_tx, Chan};
+
+use crate::coms::{DaemonIPC, IPCMessage};
+use crate::get_ipc_socket_path;
+use crate::telio_cli_interface::TelioCliInterface;
+
+/// Function to setup the DTCLI daemon's dependencies (including the async runtime) and and run it.
+///
+/// # Arguments
+///
+/// * `features` - Optional string containing the features specification for Telio in JSON.
+pub fn run_dtcli(features: Option<String>) -> Result<()> {
+    let nord_token = std::env::var("NORD_TOKEN").ok();
+    let telio_cli = TelioCliInterface::new(nord_token, features)?;
+
+    let _ = tokio::runtime::Runtime::new()?.block_on(async {
```

Comment:
We basicaly will have two tokio rutimes? one build for this and one for telio ?

Reply:
Yes, added comment with our irl discussion points to make this clearer for developers in the future.